### ISSUE TITLE
I got a bug when i parsing a pdf!!!

### ISSUE THREAD

**Issue by idiotTest** [jsvine/pdfplumber#1147]
## Describe the bug

when i use the "PDFPlumberLoader" provided by langchain,a bug occurs.
the bug is 
"File "D:\annaconda\envs\fastapitest\lib\site-packages\pdfplumber\utils\pdfinternals.py", line 16, in <genexpr>
    return "".join(PDFDocEncoding[o] for o in ords)
IndexError: string index out of range
"

## Have you tried [repairing](https://github.com/jsvine/pdfplumber/blob/stable/docs/repairing.md) the PDF?
I run the code 
```python 
pdfplumber.repair(r"D:\google\Linux运维趋势_第10期_日志分析技巧分享.pdf",
                  outfile="./repaired1.pdf",
                  gs_path=r"C:\Program Files\gs\gs10.03.1\bin\gswin64.exe")
```
but the bug is also occurs.

## Code to reproduce the problem
This is the code occur erros.
```python
from langchain_community.document_loaders import PDFPlumberLoader

loader = PDFPlumberLoader(r"./repaired1.pdf")

content = loader.load()

print(content)
```

## PDF file

this is the error file.

[Linux运维趋势_第10期_日志分析技巧分享.pdf](https://github.com/user-attachments/files/15520585/Linux._.10._.pdf)

*If applicable, add screenshots to help explain your problem.*
![image](https://github.com/jsvine/pdfplumber/assets/143317520/2225e7d9-8ef5-45ed-95b0-ecb7e49aa135)

## Environment

- Python version: 3.10 
- OS: [Linux, Windows]

**Comment by jsvine** [comment]
Hi @idiotTest, and thanks for flagging this. I've added a fix for issues like these in v0.11.1. Specific change here:  https://github.com/jsvine/pdfplumber/commit/4daf0aa1b9132a74a4d1b8960ca95b6607ac49c7

**Comment by idiotTest** [comment]
> Hi @idiotTest, and thanks for flagging this. I've added a fix for issues like these in v0.11.1. Specific change here: [4daf0aa](https://github.com/jsvine/pdfplumber/commit/4daf0aa1b9132a74a4d1b8960ca95b6607ac49c7)

ok,Thanks for your reply.I will try later.
